### PR TITLE
feat: #70 store  verifier in saved doc

### PR DESCRIPTION
**Pull request by benjaminlim00**

### CODE REVIEW

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=73

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-73

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=73

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-73

**Comment by Nebulis** [comment]
I saved Opencerts document from https://gallery.openattestation.com/, but the issuer identity is not displayed :thinking:

File: src/common/hooks/useDocumentVerifier/useDocumentVerifier.test.tsx [hunk 1]
```diff
@@ -87,4 +89,16 @@ describe("useDocumentVerifier", () => {
       overallValidity: CheckStatus.VALID
     });
   });
+
+  it("should return correct verifierType for document", async () => {
```
Comment (Nebulis) [inline src/common/hooks/useDocumentVerifier/useDocumentVerifier.test.tsx]: should you test if you pass a verifier in parameter also ?

File: src/components/DocumentList/DocumentListScreenContainer.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import React, { FunctionComponent, useState, useEffect } from "react";
-import { DocumentObject, NavigationProps } from "../../types";
+import { DocumentObject, NavigationProps, VerifierTypes } from "../../types";
 import { getData } from "@govtechsg/open-attestation";
 import { useDbContext } from "../../context/db";
 import { replaceRouteFn } from "../../common/navigation";
 import { DocumentListScreen } from "./DocumentListScreen";
 
+// TODO - get isserName from verifier instead of getData. this does not support OC documents
```
Comment (Nebulis) [inline src/components/DocumentList/DocumentListScreenContainer.tsx]: still todo ?

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=73

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-73

File: src/components/DocumentList/DocumentList.tsx [hunk 3]
```diff
@@ -22,19 +22,17 @@ export const DocumentList: FunctionComponent<DocumentList> = ({
...
-    );
-  });
+  const renderedDocumentListItem = documents.map(doc => (
+    <DocumentListItem
+      key={doc.id}
+      title={doc.title}
+      isVerified={doc.isVerified}
+      lastVerification={doc.lastVerification}
+      issuedBy={doc.issuedBy}
+      onPress={(): void => navigateToDoc(doc.id, doc.verifierType as any)} // TODO - handle typing to include undefined
```
Comment (Nebulis) [inline src/components/DocumentList/DocumentList.tsx]: TODO ..

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=73

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-73

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=73

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-73

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=73

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-73

File: src/components/DocumentList/DocumentListScreenContainer.tsx (hunk 2 above)
Comment (benjaminlim00) [reply src/components/DocumentList/DocumentListScreenContainer.tsx]: removed

**Pull request merged**